### DISCUSSION TITLE
Does Jotai allow atoms with the same debugLabel?

### DISCUSSION THREAD

**Discussion by AjaxSolutions** [pmndrs/jotai#2010] [Q&A] [answered]
I see that you can duplicate the atom debugLabel.

I think duplicating atom debugLabel should not be allowed.

They are listed twice in the developer tools, making it hard to distinguish one from the other.

**Comment by dai-shi** [comment, marked as answer]
duplicating debug labels should be allowed. (Otherwise, it's just keys which Jotai avoids in the first place.)
In some cases, we show both atom internal keys (that are unique) and debug labels.
